### PR TITLE
TA4_7

### PR DESCRIPTION
**Pull request by eszterkoczak**
Summary of changes and the sender team

The irrelevant parts must be removed!

# Fixed bugs (with referenced issue)

 - Egocar now turns left pressing left arrow #ISSUE-ID
 - ...

# Added functionalities (with referenced issue)

 - Flux capacitor has been implemented #ISSUE-ID
 - ...

# (Breaking) changes (with referenced issue)

 - getWorldObjectReferencePoint no longer returns the top-left corner
   but the centroid #ISSUE-ID
 - ...

# Other (test, documentation addition, refactor)

 - e.g. Tests added to powertrain
 - Input handling was refactored
 - ...

### CODE REVIEW

File: src/main/java/hu/oe/nik/szfmv/common/DynamicMoving.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public DynamicMoving() {
+        acceleration = 0;
+        angle = 0;
+        speed = 0;
+        vector = new Point(0,0);
+    }
+
+    public void CalculateNewVector(int acceleration){
+        speed += acceleration;
+        double road = ((double)speed/3600)*1000;
```
Comment (hannalegar) [inline src/main/java/hu/oe/nik/szfmv/common/DynamicMoving.java]: magic numbers: 3600, 1000

File: src/test/java/hu/oe/nik/szfmv/common/DynamicMovingTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Test
+    public void testNewVectorCoordinates(){
+        moving = new DynamicMoving();
+
+        Point oldPoint = moving.getVector();
+        moving.CalculateNewVector(100);
+
+        assertNotEquals( oldPoint , moving.getVector());
```
Comment (hannalegar) [inline src/test/java/hu/oe/nik/szfmv/common/DynamicMovingTest.java]: két felesleges space, oldPoint előtt és után

Comment (galbavi) [reply src/test/java/hu/oe/nik/szfmv/common/DynamicMovingTest.java]: kell rá egy olyan, hogy Code/Reformat Code és az megcsinál minden ilyet

Comment (pintergreg) [reply src/test/java/hu/oe/nik/szfmv/common/DynamicMovingTest.java]: Esetleg [feltenni és beállítani a Save actions plugint](https://szfmv2018-osz.github.io/handout/kodformazas/#save-actions). Automatizáljuk amit lehet... :wink:

**Pull request merged**